src/main/kotlin/org/rust/ide/refactoring/inlineUtils.kt

```diff
@@ -82,8 +82,7 @@ class RsInlineUsageViewDescriptor(val element: PsiElement, val header: String) :
             UsageViewBundle.getOccurencesString(usagesCount, filesCount))
 
     override fun getCodeReferencesText(usagesCount: Int, filesCount: Int) =
-        RefactoringBundle.message("invocations.to.be.inlined",
-            UsageViewBundle.getReferencesString(usagesCount, filesCount))
+        "Invocations to be inlined " + UsageViewBundle.getReferencesString(usagesCount, filesCount)
```

Comment:
Why do you drop using message bundle here?

Reply:
<img width="1254" alt="Screenshot 2021-03-24 at 19 22 12" src="https://user-images.githubusercontent.com/6079006/112345632-4432c500-8cd6-11eb-8542-618621ca7210.png">

Reply:
Interesting. Actually, the platform contains this property but in `IdeDeprecatedMessagesBundle.properties`, i.e. the message was deprecated in the platform and moved from `RefactoringBundle` to `IdeDeprecatedMessagesBundle` not to break plugins. Let's keep it as is and fix it later during localization process
For now you can just suppress it